sdk/hdinsight/Microsoft.Azure.HDInsight.Job/tests/Microsoft.Azure.HDInsight.Job.Tests.csproj

```diff
@@ -23,4 +23,10 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
This is a 2MB binary file that is added to this repo. I would recommend not checking it in.

@weshaggard to see if there are ways of not checking in the file.

Reply:
That is a spark job jar packet for testing sunmitting feature. If it should not be ckecked in, I will find another way to get it during sceniro tests. Thanks for your recommendation. I will update code ahout that.

Reply:
Yes if possible we should try to avoid checking jar files which are binary files and can cause bloat overtime to the repo history.